src/FieldDefinition.hack

```diff
@@ -1,13 +1,12 @@
 namespace Slack\GraphQL;
 
-use namespace HH\Lib\C;
+use namespace HH\Lib\{C, Vec};
 
 type ArgumentDefinition = shape(
```

Comment:
This is mostly the same as `Introspection\__InputValue` except in `Introspection\__InputValue` `default_value` is camel case and is a string value. This duplication seems worth it to keep `Introspection\__InputValue` a shape

Reply:
We can probably get rid of this and use your version everywhere. Validation only cares about the existence of a default value so it doesn't care if it's string or mixed.